tests/testthat/test_NAN.R

```diff
@@ -14,3 +14,44 @@ fit <- cv.iregnet(
 fit <- cv.iregnet(
   realNAN$feature.mat, realNAN$target.mat,
   family="gaussian")
+
+# Custom data which works fine
```

Comment:
All these `y`s seem to be dangling here with no purpose :)

Are these meant for some tests? Could you please add them into `test_that` tests?

Reply:
Like we discussed before, for cross validation, calling iregnet again gives warning messages for each fold. So there are like 5 warning messages of "Failed to converge, try adding more data" and I didn't know how to handle that. I've left it there for future commits when I'll improve the error and warning messages for CV.